app/models/course/condition/level.rb

```diff
@@ -14,4 +14,8 @@ def title
   def satisfied_by?(course_user)
     course_user.level_number >= minimum_level
   end
+
+  def self.required_classes
+    []
```

Comment:
It's because `Course::Condition::Level` doesn't depend on any class, unlike `Course::Condition::Achievement` depends on `Course::Achievement`.